src/open_prime_hunters_rando/string_tables/string_tables.py

```diff
@@ -210,14 +210,25 @@ def get_group_max_string_id(self, string_group: str) -> str:
         string_id = "100"
         for string in self.strings:
             if string.string_id[-1] == string_group:
-                string_id = string.string_id
+                string_id = string.string_id[:-1]
         return string_id
 
-    def append_string(self, string_group: str, template: StringEntry) -> str:
+    def reverse_string(self, string: str) -> str:
+        return string[::-1]
+
+    def append_string(self, string_group: str, template: StringEntry) -> None:
+        # Calculate the max string id of that string group
```

Comment:
can you add a quick comment about the general structure of the string?
These comments are fairly useless if you don't know that they function á la ZYXA